android/assets/jsons/translations/template.properties

```diff
@@ -815,6 +816,8 @@ Combat =
 Experience = 
 Pillaging = 
 Research Agreements = 
+Great People = 
```

Comment:
Doesn't seem to appear anywhere in the UI?

Reply:
Ahhhh I see 
We need to translate tutorial names
Got it, this should be automatic